stdlib/2and3/builtins.pyi

```diff
@@ -575,7 +575,9 @@ if sys.version_info >= (3,):
         def endswith(self, suffix: Union[bytes, Tuple[bytes, ...]]) -> bool: ...
         def expandtabs(self, tabsize: int = ...) -> bytes: ...
         def find(self, sub: Union[bytes, int], start: Optional[int] = ..., end: Optional[int] = ...) -> int: ...
-        if sys.version_info >= (3, 5):
+        if sys.version_info >= (3, 8):
+            def hex(self, sep: Union[str, bytes] = ..., bytes_per_sep: int = ...) -> str: ...
```

Comment:
Fun fact: if you pass a negative number for `bytes_per_sep`, it uses the absolute value.

Reply:
Yes, but:

> Positive values calculate the separator position from the right, negative values from the left.

https://github.com/python/cpython/blob/master/Python/pystrhex.c#L88